ROADMAP.md

```diff
@@ -1,7 +1,45 @@
 # KF Serving 2019 Roadmap
-## Q2 2019
+## Q3 2019
+### v0.3 Performance and Stability (ETA: September 1, 2019)
+Objective: "Prevent feature regressions with 80% e2e test coverage against a live Cluster."
+* Automated End-to-End Tests
+    * Execute against a Kubeflow maintained GKE Cluster.
+    * Achieve >80% Test Coverage of Supported Features.
+
+Objective: "Prevent performance regressions across a known set of representative models."
+* Automated Performance Tests 
+    * Define a set of Models to test covering a wide array of usecases and frameworks.
+    * Publish performance results in a temporally comparable way.
+
+Objective: "Improve the Serverless Experience by reducing cold starts/stops to 10 seconds on warmed models."
+* Model Caching
+    * Reduce model download time by caching models from cloud storage on Persistent Volumes.
+* Image Caching
+    * Reduce container download time by ensuring images are cached in all cloud environments.
+* Server Shutdown
+    * Ensure that all model servers shutdown within 10 seconds of not receiving traffic.
+
+### v0.2 Integrate with the ML Ecosystem (ETA: August 15, 2019)
+Objective: "Continue to simplify the user experience by deeply integrating with the Kubeflow Ecosystem."
+* Kubeflow Integration
+    * Prepare KFServing to release v0.2 and v0.3 alongside Kubeflow 0.7.
+    * Integrate with `kfctl generate` and `kfctl apply`.
+    * Deploy as a [Kubernetes Application](https://github.com/kubernetes-sigs/application).
+    * Integrate with PVCs to enable integration with Kubeflow Pipelines
+    * Integrate with Fairing to enable deployment from a Notebook
+
+Objective: "Empower users to deeply understand their predictions and validate KFServing's static graph architecture."
+* Explainability
+    * Deploy a predictor and explainer, powered by Alibi.
+    * Deploy a predictor and explainer, powered by user specified explainer container.
 
-### Core CUJs
+Objective: "Increase coverage of ML frameworks to support previously unsupported customer workloads."
+* Frameworks
```

Comment:
We also need to support scikit-learn IMO. It is also fairly easy to support with existing scaffolding

Reply:
Already supported